web_responsive/models/res_users.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018-2019 Alexandre Díaz
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    chatter_position = fields.Selection([
+        ('normal', 'Normal'),
+        ('sided', 'Sided'),
+    ], string="Chatter Position", default='normal')
```

Comment:
```suggestion
    ], string="Chatter Position", default='sided')
```

I think it's time to flip the switch...